.github/workflows/ci.yml

```
@@ -0,0 +1,24 @@
+name: Cargo Build & Test
```

Since the file is named `ci.yml` this should include all potential CI steps, not just Cargo.

```suggestion
name: Continuous integration
```

Alternatively, rename this file to `cargo.yml` and just shorten the name here:

```suggestion
name: Cargo
```